django/cantusdb_project/main_app/templates/source_detail.html

```diff
@@ -34,15 +34,20 @@ <h3>{{ source.heading }}</h3>
 {% endif %}
 
 <dl>
+    <dt>Cantus Siglum</dt>
+    <dd>{{ source.short_heading }}{% if source.name %} <i>"{{ source.name }}"</i>{% endif %}</dd>
```

Comment:
The point of the `heading` and `short_heading` methods is to make display consistent across the site. So if the name should be displayed with the short heading, it should probably go in the helper method, and not on a case-by-case basis in the templates.

Reply:
I'm not sure it should in all cases?...in any case, realizing there are a few different persistent source display discrepancies, this among them, so will ask Debra.